client/my-sites/plugins/marketplace/components/simulated-progressbar/index.tsx

```diff
@@ -34,7 +34,7 @@ export function resolveStep(
 }
 
 const SIMULATION_REFRESH_INTERVAL = 2000;
-const INCREMENTED_PERCENTAGE_SIZE_ON_STEP = 6;
+const INCREMENTED_PERCENTAGE_SIZE_ON_STEP = 2;
```

Comment:
Time to install takes longer than expected (Close to 1.5 minutes) so decreased the step size.

Reply:
Won't this change in different circumstances and network/device specifics? Also, is the necessity to tweak this a signal that we need to find a better way to simulate or implement the progress bar?

Reply:
We definitely need to find a better way to simulate this. But right now I am not sure of a quick way to do this. I did made a small change to make at least speed it up once the work is done. 
https://github.com/Automattic/wp-calypso/blob/add/atomic-site-free-plugin-flow/client/my-sites/plugins/marketplace/components/simulated-progressbar/index.tsx#L40-L44